src/Tabs.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module TabsListTab
+ *
+ * Direct child of `TabsList`.
+ * Renders individual tab control.
+ * Accepts a `selected` prop.
+ */
+export const TabsListTab = React.createClass({
```

Comment:
This was a style choice for tabs because this is a composite component - it was easier for me to look at `propTypes` inside of the object literals. Now that I'm looking at this again, I don't think that's a compelling enough argument to break from standard convention. I'll change this back.